php/Terminus/Helpers/UpdateHelper.php

```diff
@@ -17,11 +17,13 @@ class UpdateHelper extends TerminusHelper
     public function getCurrentVersion()
     {
         $request  = new Request();
-        $url      = 'https://api.github.com/repos/pantheon-systems/terminus/releases';
-        $url     .= '?per_page=1';
-        $response = $request->request($url, ['absolute_url' => true]);
-        $release  = array_shift($response['data']);
-        $cache    = new FileCache();
+        $url = 'https://api.github.com/repos/pantheon-systems/terminus/releases/latest';
+        try {
+            $release = $request->request($url, ['absolute_url' => true,])['data'];
+        } catch (\Exception $e) {
+            return;
```

Comment:
I think returning `null` here may have unexpected consequences when the sending method tries to compare the output. It also breaks the contract of the return value... and still will log something: https://github.com/pantheon-systems/terminus/blob/6ff2ddcae938b8efa8fcf836e6239a0fb378d741/php/Terminus/Helpers/UpdateHelper.php#L48-L60